backend/service/db.go

```diff
@@ -115,6 +118,18 @@ func prepareStatements(ctx context.Context, connection *pgx.Conn) error {
 	if err != nil {
 		return err
 	}
+	_, err = connection.Prepare(ctx, "select_all_buildings", 
```

Comment:
If we had more than a couple of buildings this procedure shouldn't exist. Of course we only have like 30 or 40 of them but this can get unwieldy _very, very_ quickly if everyone is hammering this particular routine. Since buildings don't change all that much, should we (further up the call stack) introduce a cached response that minimize the risk of overloading our pool?

Also, presumably, we'd implement rate limiting, which would also use redis : )

Reply:
Sounds good but likely for another PR